docs/topics/migrations.txt

```diff
@@ -476,6 +476,30 @@ You can pass a second callable to
 want executed when migrating backwards. If this callable is omitted, migrating
 backwards will raise an exception.
 
+.. note::
+
+    When migrating data across models spanning more than one app, ``dependencies``
+    should include the latest migration of each app that is involved, otherwise you
+    will get an error: "LookupError: No installed app with label '``myappname``'".
```

Comment:
I confirmed what Markus said in an earlier comment. This seems no longer the case in 1.8+. I tested by adding a data migration in the tutorial and having it access `apps.get_model('auth', 'User')`. It's a LookupError on 1.7, but works fine in 1.8.